static/app/components/events/eventReplay/replayInlineOnboardingPanelBackend.tsx

```diff
@@ -0,0 +1,72 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import replaysInlineOnboarding from 'sentry-images/spot/replay-onboarding-backend.svg';
+
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {EventReplaySection} from 'sentry/components/events/eventReplay/eventReplaySection';
+import PageBanner from 'sentry/components/replays/pageBanner';
+import platforms, {otherPlatform} from 'sentry/data/platforms';
+import {IconBroadcast} from 'sentry/icons/iconBroadcast';
+import {t, tct} from 'sentry/locale';
+import {PlatformKey} from 'sentry/types';
+import {useReplayOnboardingSidebarPanel} from 'sentry/utils/replays/hooks/useReplayOnboarding';
+
+type OnboardingCTAProps = {
+  platform: PlatformKey;
+};
+
+export default function ReplayInlineOnboardingPanelBackend({
+  platform,
+}: OnboardingCTAProps) {
+  const [isHidden, setIsHidden] = useState(false);
+  const {activateSidebar} = useReplayOnboardingSidebarPanel();
+
+  const platformName = platforms.find(p => p.id === platform) ?? otherPlatform;
+
+  return (
+    <EventReplaySection
+      actions={
+        <Button borderless onClick={() => setIsHidden(!isHidden)}>
+          {t('Hide Details')}
+        </Button>
+      }
+    >
+      <Fragment>
+        {isHidden ? null : (
+          <PageBanner
+            button={
+              <ButtonBar gap={1}>
+                <Button priority="default" onClick={() => {}}>
+                  {t('View Sample')}
+                </Button>
```

Comment:
we can bring in the same sample replay button from the list page:

put this outside the component:
```
const OnboardingCTAButton = HookOrDefault({
  hookName: 'component:replay-onboarding-cta-button',
  defaultComponent: null,
});
```

and right here it's:
```suggestion
                <OnboardingCTAButton />
```